.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
Please .gitignore the .vscode folder.

Reply:
you'll need to delete this folder and push that deletion to git before the .gitignore will work for it